app/models/user.rb

```diff
@@ -7,6 +7,8 @@ class User < ApplicationRecord
 
   before_validation :generate_verification_code, on: :create
 
+  has_many :rents, dependent: :destroy
```

Comment:
Same here, do you think if a user is deleted that the rent should be deleted too?